### PR TITLE
added cart to checkout

### PR DESCRIPTION
**Pull request by compassface**

### CODE REVIEW

**Comment by joshuarichard** [comment]
found lint :(

**Comment by joshuarichard** [comment]
👎 

screen shot 2016-04-17 at 8 16 03 pm
screen shot 2016-04-17 at 8 15 11 pm
screen shot 2016-04-17 at 8 15 19 pm

**Comment by compassface** [comment]
dude, the entire bar is messed up

**Comment by compassface** [comment]
image

File: public/views/children.html [hunk 1]
```diff
@@ -46,7 +46,10 @@
                     <div class="col-sm-2 pre-active" style="text-align: center" >
                         <a style="text-align: center"  href="/views/children.html">Apadrina</a>
                     </div>
-                    <div class="col-sm-2 pre-active">
+                    <div class="col-sm-1" style="text-align: center" >
```
Comment (joshuarichard) [inline public/views/children.html:49]: you should make this col-md-2 and add the pre-active class so that it's orange and not blue

**Comment by joshuarichard** [comment]
success! no lint!

File: public/views/children.html (hunk 1 above)
Comment (joshuarichard) [inline public/views/children.html:49]: actually, maybe if you change all three to col-md-1 the bar will all be on one line like it should be

**Comment by compassface** [comment]
image

**Comment by compassface** [comment]
rly?

**Comment by joshuarichard** [comment]
push it and i'll see, remember your screen is a higher resolution than jake and mine

**Comment by jacobbuettner** [comment]
So the header is still a little messed up, idk what happened to the home button
image

**Comment by jacobbuettner** [comment]
I I would recommend we change it to this
image

**Comment by jacobbuettner** [comment]
Also, the change doesn't seem to work...
Need to add an if statement at line 166 that will initialize carte to 0 if sessionStorage.getItem('cart') == null
image

**Comment by jacobbuettner** [comment]
JS needs a good amount of rework as well

**Pull request closed without merging**